docs/Tutorial.md

```diff
@@ -850,7 +850,7 @@ For instance, let's imagine you have to use the `my.api.url` REST API, which exp
 
 | Action              | Expected API request                                                                    |
 | ------------------- | --------------------------------------------------------------------------------------- |
-| Get list            | `GET http://my.api.url/posts?sort=["title","ASC"]&range=[0, 24]&filter={"title":"bar"}` |
+| Get list            | `GET http://my.api.url/posts?sort=title&order=ASC&page=1&perPage=50&filter={"title":"bar"}` |
```

Comment:
No, this doesn't match the dataProvider example below.

Reply:
Not sure what's wrong. Is it the number of items per page? (See next commit)

Reply:
Check the example dataProvider for getList starting at L876

Reply:
Oh I see I've mistaken the URL of the list view page in the React-admin app (which should approx. look like what I'm proposing) with the URL of the call to the API. My bad :sweat_smile: